src/test/integration/IntegrationDeployer.t.sol

```diff
@@ -79,85 +80,7 @@ abstract contract IntegrationDeployer is ExistingDeploymentParser {
     // Constants
     uint64 constant MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR = 32e9;
 
-    IStrategy constant BEACONCHAIN_ETH_STRAT = IStrategy(0xbeaC0eeEeeeeEEeEeEEEEeeEEeEeeeEeeEEBEaC0);
-    IERC20 constant NATIVE_ETH = IERC20(0xbeaC0eeEeeeeEEeEeEEEEeeEEeEeeeEeeEEBEaC0);
-
-    uint constant MIN_BALANCE = 1e6;
-    uint constant MAX_BALANCE = 5e6;
-    uint constant GWEI_TO_WEI = 1e9;
-
-    // Paused Constants
```

Comment:
I'm seeing these paused constants were deleted from the Integration tests. Noting that we may need this later for fork tests to simulate unpausing certain features or also simulating the EigenPod migration.